assets/translations/en.json

```diff
@@ -168,8 +168,9 @@
     "enter_email_mess": "Please enter your email address",
     "invalid_email": "Invalid email address",
     "email_address": "Email address",
-    "email_sent": "Reset email has been sent",
-    "email_send_error": "That email does not exist in the system.",
+    "email_sent_1": "Reset email has been sent to",
+    "email_sent_2": "if the account exists.",
+    "email_send_error": "Unable to send reset email at this time.",
```

Comment:
Maybe, we should move it from "forgot_pass_form" into a separate group of keys?

Reply:
as a suggestion from @farmerpaul I'm thinking to keep it as 
```
{
  "email_send_success": "Reset email has been sent to <strong>{{email}}</strong> if the account exists.",
}
```